continuous-integration/travis/install_deps.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
Is this script needed? Shouldn't `re2` already be installed via Homebrew and Apt on macOS and Ubuntu according to `.travis.yml`?

Reply:
This isn't run in the Ubuntu build. I ran into issues with Homebrew's installation of re2 on macOS.